### ISSUE TITLE
Deprecated UIWebView Apple warning

### ISSUE THREAD

**Issue by harshilJs** [zeddz92/react-native-paypal#14]
Can anyone confirm if this library uses the UIWebView class or not? I'm getting warning from apple warning after installing this library. I'm using v1.0.9 of this lib and react-native v0.61.4

**Comment by harshilJs** [comment]
getting this after running `grep -r UIWebView ./*`
.
<img width="634" alt="Screenshot 2020-05-21 at 4 22 29 PM" src="https://user-images.githubusercontent.com/49385977/82552251-61ab7580-9b7f-11ea-95ea-f93dc9e54ce2.png">

these are the pod versions I'm using
```
 - PayPal-iOS-SDK (2.18.1):
    - PayPal-iOS-SDK/CardIO (= 2.18.1)
    - PayPal-iOS-SDK/Core (= 2.18.1)
  - PayPal-iOS-SDK/CardIO (2.18.1):
```

**Comment by jothikannan** [comment]
Yes I am also having the same issue, @harshilJs have you found any solution?

**Comment by new-furry** [comment]
Hi everyone.
Also, I am getting same error...
anyone solved this probem?
If this is not solved, then which library should i use for paypal?
If anyone help me, I am very appreciate.

**Comment by harshilJs** [comment]
hey @jothikannan and @new-furry, unfortunately due to lack of time we had to continue by putting webview of paypal page and we're making payment by passing parameters to web URL, but we're finding solutions for this warning too, Lemme know if you guys find anything. thanks.

**Comment by new-furry** [comment]
Hi.
for iOS, this library doesn't work.
So I have found one solution during google search.
https://medium.com/zestgeek/paypal-integration-in-react-native-9d447df4fce1
If you follow this guide, you can understand and implement paypal integration.
hope this helps anyone.

**Comment by harshilJs** [comment]
@new-furry that's exactly what we did. thanks for sharing it here.
